src/components/pages/roster-page/edit-roster-entry-dialog.tsx

```diff
@@ -82,7 +82,7 @@ export const EditRosterEntryDialog = (props: EditRosterEntryDialogProps) => {
     updateRosterEntryProperty('unit', event.target.value);
   };
 
-  const getUnitName = (unitId: string) => {
+  const getUnitName = (unitId: number) => {
```

Comment:
This function always seems to return 'Unknown' when testing locally. Change seems to occur correctly on backend (actually changing unit assigned in roster), but dialog always pops up 'Are you sure you want to move individual to Unknown?' Not sure if it that is a factor of our seeding in local build or not.

Reply:
This was caused because the incoming unit ID was actually a string due to an incorrect handling of the unit changed event.